### ISSUE TITLE
Uninstall option in Windows, etc.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dequis/msn-pecan#131]
```
What steps will reproduce the problem?

The installation of msn-pecan-0.0.17.exe isn't listed in Add or Remove
Programs in Windows.

What is the expected output? What do you see instead?

It would be useful to have some option to uninstall msn-pecan, either by
deleting a single file, or by having it listed in Add or Remove Programs.
It could potentially be listed "Pidgin - msn-pecan 0.0.17 Add-on", etc.

What version of the product are you using? On what operating system?

0.0.17 on Windows XP x64
```

Original issue reported on code.google.com by `abbconsu...@gmail.com` on 16 Feb 2009 at 12:07

**Comment by GoogleCodeExporter** [comment]
```
You can manually uninstall msn-pecan by removing libmsn-pecan.so. However, it 
would
be nice to have an uninstaller.
```

Original comment by `felipe.contreras` on 16 Feb 2009 at 12:52
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**, **Priority-High**, **OpSys-Windows**

**Comment by GoogleCodeExporter** [comment]
```
Until an uninstaller is available, it would be useful to have this info included
somewhere in the wiki.

Note to Windows users: libmsn-pecan.dll is in the \Pidgin\plugins\ folder.
```

Original comment by `abbconsu...@gmail.com` on 16 Feb 2009 at 1:38

**Comment by GoogleCodeExporter** [comment]
Original comment by `felipe.contreras` on 13 Dec 2009 at 11:55
- Added labels: **Milestone-0.2.0**

**Comment by GoogleCodeExporter** [comment]
```
I just implemented this, will be there on 0.1.0-rc3.
```

Original comment by `felipe.contreras` on 27 Jan 2010 at 12:35
- Changed state: **Fixed**
- Added labels: **Milestone-0.1.0**
- Removed labels: **Milestone-0.2.0**
